backend/corpora/common/utils/cxg_generation_utils.py

```diff
@@ -27,7 +27,7 @@ def convert_dictionary_to_cxg_group(cxg_container, metadata_dict, group_metadata
     # array.
     tiledb.from_numpy(array_name, np.zeros((1,)))
 
-    with tiledb.DenseArray(array_name, mode="w") as metadata_array:
+    with tiledb.open(array_name, mode="w") as metadata_array:
```

Comment:
out-of-scope for the refactoring, but should this specify an explicit `ctx` arg?